snippets/header-drawer.liquid

```diff
@@ -183,7 +183,7 @@
                   </localization-form>
                 {% endif %}
 
-                {%- if section.settings.enable_language_selector and localization.available_languages.size > 1 -%}
+                {%- if localization.available_languages and localization.available_languages.size > 1 -%}
```

Comment:
I'm seeing some styling issues in the drawer, might have been due to the change in class to `header__localization` 🤔

Reply:
The error arose when we removed the header__icons--localization as a class. Since there is another class with it's own styling  called header__localization, I think we needed the header__icons--localization one to make the popup appear above in the menu drawer instead of cut off like the bug shown above